src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,14 @@
+$clock-size: 500px;
+$clock-radius: $clock-size / 2;
+$center-width: $clock-size / 25;
+
+$minute: 120s;
```

Comment:
Are you sure that the minute hand makes a full revolution in 120 seconds?

Reply:
just wanted to show that it is moving, i know that minute hand makes a full rotation during 3600s